agent/util-scripts/gold/test-client-tool-meister/test-53.txt

```diff
@@ -132,4 +120,8 @@ INFO pbench-tool-meister main -- Remove pid file ... (tm-default-testhost.exampl
 /var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/pidof -x iostat
 /var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/pidof -x mpstat
 /var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/pidof -x mpstat
```

Comment:
I thought the `test-bin/pidof` was deleted above: so how come we have it running here still?

Reply:
nm - it just got moved.